AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -428,6 +428,8 @@ sonichits.com,thepostmillennial.com,muropaketti.com,visajourney.com,dennikn.sk,i
 @@||di.se/scripts/advertisement__*.js
 !#safari_cb_affinity
 !
+mc-at.org#%#//scriptlet("prevent-setTimeout", "showads", "7000")
+mc-at.org#@#.showads
```

Comment:
This rule must be wrapped into` !#safari_cb_affinity(general)` directive

Please read about it here:
`https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#pre-processor-directives`

Reply:
@Sergey-Lyapin I actually don't understand the Python's `If directive` and the hints. Hints like `!+  !+ PLATFORM(ios, ext_android_cb, ext_safari)`. What's the difference and when should I use the correct one?

Reply:
@wout-codes
"Platform" affects only one rule under the hint, but we can wrap several rules into if/endif - this is the main difference =)
Use "Platform" if it must affect only one rule